src/ILLink.Tasks/CreateRuntimeRootDescriptorFile.cs

```diff
@@ -356,7 +356,7 @@ static void AddXmlTypeNode (XmlDocument doc, XmlNode assemblyNode, string typeNa
 					typeNode.Attributes.Append (preserve);
 				} else if ((members.fields == null) && (members.methods == null)) {
 					XmlAttribute preserve = doc.CreateAttribute ("preserve");
-					preserve.Value = "nothing";
+					preserve.Value = "none";
```

Comment:
It's correct to have only one between the two, but just be aware that seems like runtime is using the 'nothing' notation since is what it was documented. See [file](https://github.com/dotnet/runtime/blob/main/src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml#L59).
The way I see it we either agreed on nothing or have to change other files outside the linker repo who may have used 'nothing' already

Reply:
Seems like the code is looking for Nothing and if other value is passed then it defaults to Nothing
https://github.com/dotnet/linker/blob/main/src/linker/Linker.Steps/DescriptorMarker.cs#L137-L146